packages/rate-limit/src/rate-limit.ts

```diff
@@ -135,6 +147,7 @@ export function rateLimit(options?: Partial<RateLimitOptions>) {
 
       next()
     } catch (e) {
+      console.log(e)
```

Comment:
not sure if we need to put `console.log` here

u can add `console.log` to `onError`